site/content/api/Tenant-API.md

```diff
@@ -376,6 +377,12 @@ Protocol adapters SHOULD use the configuration properties set for a tenant when
 
 The JSON object MAY contain an arbitrary number of additional members with arbitrary names of either scalar or complex type.
 
+## Limits Configuration Format
+The table below provides an overview of the members defined for the *limits* JSON object:
+
+| Name                     | Mandatory | Type          | Default Value | Description |
+| :------------------------| :-------: | :------------ | :------------ | :---------- |
+| *max-connections*        | *no*      | *long*      | *-1*          | The maximum number of connections to devices to be allowed. Default value -1 indicates that no limit is set.|
```

Comment:
I don't think that there is a type *long* in JSON, is there? So this should probably be *number*.

Reply:
I have changed to _number_